docs_user/modules/proc_adopting-autoscaling.adoc

```diff
@@ -72,10 +53,27 @@ openstack alarm create --name high_cpu_alarm \
 --threshold 7000000000
 ----
 
-* Verify the state of alarm
+.Verification
+
+. If autoscaling services are enabled, inspect the aodh pods:
++
```

Comment:
Wouldn't autoscaling services be enabled already?

Reply:
> Wouldn't autoscaling services be enabled already?

During verification we can safely assume that autoscaling was enabled, so it could just say `Inspect Aodh pods:`